src/portal/src/i18n/lang/zh-tw-lang.json

```diff
@@ -1420,427 +1419,427 @@
+    "NAME_REQUIRED": "名稱為必填項",
+    "PROVIDER_REQUIRED": "提供者為必填項",
+    "ADDED_SUCCESS": "成功新增政策",
+    "UPDATED_SUCCESS": "成功更新政策",
+    "EXECUTE": "執行",
+    "EXECUTE_SUCCESSFULLY": "成功執行",
+    "EXECUTE_TITLE": "確認政策執行",
+    "EXECUTE_SUMMARY": "您是否要執行策略 {{param}}？",
+    "STOP_TITLE": "確認停止執行",
+    "STOP_SUMMARY": "您是否要停止執行策略 {{param}}？",
+    "STOP_SUCCESSFULLY": "成功停止執行",
+    "STATUS_MSG": "狀態訊息",
+    "JOB_PLACEHOLDER": "找不到任何執行作業",
+    "PROVIDER_TYPE": "供應商",
+    "ID": "執行 ID",
+    "NO_PROVIDER": "請先新增一個供應商",
     "ARTIFACT": "Artifact",
     "DIGEST": "Digest",
-    "TYPE": "Type",
-    "TASKS": "Tasks",
-    "TASKS_PLACEHOLDER": "We couldn't find any task",
-    "SEVERITY_WARNING": "Vulnerability settings here conflicts with the relevant project configuration that will override the settings here",
-    "REPOS": "Repositories",
-    "TAGS": "Tags",
-    "CRITERIA": "Criteria",
-    "ONLY_SIGNED": "Only signed images",
-    "PREHEAT_ON_PUSH": "Preheat on push",
-    "START_TEXT": "No vulnerability severity of",
-    "EDN_TEXT": "and above",
-    "LABELS": "Labels",
-    "SCHEDULE": "Schedule",
-    "TEST_FAILED": "Test failed",
-    "MANUAL": "Manual",
-    "SCHEDULED": "Scheduled",
-    "EVENT_BASED": "Event based",
-    "EVENT_BASED_EXPLAIN_LINE1": "The policy will be evaluated whenever the following events are occurred:",
-    "EVENT_BASED_EXPLAIN_LINE2": "Artifact is pushed",
-    "EVENT_BASED_EXPLAIN_LINE3": "Artifact is labeled",
-    "EVENT_BASED_EXPLAIN_LINE4": "Artifact is scanned",
-    "REPO_REQUIRED": "Repository is required",
-    "TAG_REQUIRED": "Tag is required",
-    "DELETE_SUCCESSFULLY": "Deleted policy successfully",
-    "UPDATED_SUCCESSFULLY": "Updated policy successfully",
-    "EXECUTIONS": "Executions",
-    "TAG_SEPARATOR": "Enter multiple comma separated tags,tag*,or **",
-    "CONTENT_WARNING": "Content trust settings here conflicts with the relevant project configuration that will override the settings here",
-    "PREHEAT_EXPLAIN": "Preheat will migrate the image to the p2p network",
-    "CRITERIA_EXPLAIN": "As specified in 'Deployment security' section under Configuration tab",
-    "SKIP_CERT_VERIFY": "Check this box to skip certificate verification when the remote provider uses a self-signed or untrusted certificate.",
-    "NAME_TOOLTIP": "Policy name should be at least 2 characters long with lower case characters, numbers and ._- and must be start with characters or numbers.",
-    "NEED_HELP": "Please ask your system admin to add a provider first"
+    "TYPE": "類型",
+    "TASKS": "任務",
+    "TASKS_PLACEHOLDER": "找不到任何任務",
+    "SEVERITY_WARNING": "此處的漏洞設定與相關專案配置衝突，將覆蓋此處的設定",
+    "REPOS": "儲存庫",
+    "TAGS": "標籤",
+    "CRITERIA": "標準",
+    "ONLY_SIGNED": "僅限已簽名的映像檔",
+    "PREHEAT_ON_PUSH": "推送時預熱",
+    "START_TEXT": "無漏洞嚴重性",
+    "EDN_TEXT": "及以上",
+    "LABELS": "標籤",
+    "SCHEDULE": "排程",
+    "TEST_FAILED": "測試失敗",
+    "MANUAL": "手動",
+    "SCHEDULED": "已排程",
```

Comment:
This item is used to describe the task type.
<img width="261" alt="image" src="https://user-images.githubusercontent.com/30999793/236374119-70f1af10-25e3-45c6-91f2-170cb0507b67.png">

Reply:
Updated, thanks.